### PR TITLE
implement #69 use classnames module to construct react classnames

### PR DESCRIPTION
**Pull request by underbyte**
#69

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,6 +41,7 @@
   "dependencies": {
     "babel-plugin-transform-runtime": "6.4.3",
     "babel-runtime": "6.3.19",
+    "classnames": "^2.2.3",
```
Comment (adampickeral) [inline package.json]: exact version

**Comment by adampickeral** [comment]
better commit message:

```
refactor(classnames): use classnames to build css classes

resolves issue #69
```

**Comment by j00k** [comment]
:rocket:

**Comment by rogeriochaves** [comment]
Cool, I like that :+1:

**Pull request merged**